bundle/tests/scorecard/kuttl/semeru/00-assert.yaml

```diff
@@ -32,14 +32,7 @@ spec:
       serviceAccountName: my-wsliberty-app
       schedulerName: default-scheduler
       containers:
-        - resources:
```

Comment:
@yongja79 It'll be good to keep the CPU and memory request and limit checks. The values just need to be updated to this (to match default values set by WLO):

```
        - resources:
            limits:
              cpu: '2'
              memory: 1200Mi
            requests:
              cpu: 100m
              memory: 800Mi
```